src/components/layout/Layout.tsx

```diff
@@ -33,8 +33,9 @@ const Layout = (props: LayoutProps) => {
       alert('This browser does not support desktop notification. We suggest using a different browser.');

```

Comment:
Can you change this to just return here instead of alerting. The spam very annoying on browsers like IOS Safari